server.js

```diff
@@ -15,7 +15,7 @@ import express from 'express';
 import dotenv from 'dotenv'
 require('dotenv').config();
 
-const LOCAL_PORT = 8081;
+const LOCAL_PORT = 8080;
```

Comment:
Not really sure why this was changed but it shouldn't impact much of the app regardless.